src/datashare/azext_datashare/generated/_params.py

```diff
@@ -0,0 +1,307 @@
+        c.argument('properties_target_email', help='The email the invitation is directed to.')
+        c.argument('properties_target_object_id', help='The target user or application Id that invitation is being sent to. Must be specified along TargetActiveDirectoryId. This enables sending invitations to specific users or applications in an AD tenant.')
+
+    with self.argument_context('datashare invitation delete') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('account_name', help='The name of the share account.')
+        c.argument('share_name', help='The name of the share.')
+        c.argument('invitation_name', help='The name of the invitation.')
+
+    with self.argument_context('datashare share list') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('account_name', help='The name of the share account.')
+        c.argument('skip_token', help='Continuation token')
+
+    with self.argument_context('datashare share show') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('account_name', help='The name of the share account.')
+        c.argument('share_name', help='The name of the share.')
+
+    with self.argument_context('datashare share create') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('account_name', help='The name of the share account.')
+        c.argument('share_name', help='The name of the share.')
+        c.argument('properties_description', help='Share description.')
+        c.argument('properties_share_kind', arg_type=get_enum_type(['CopyBased', 'InPlace']), help='Share kind.')
+        c.argument('properties_terms', help='Share terms.')
+
+    with self.argument_context('datashare share delete') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('account_name', help='The name of the share account.')
+        c.argument('share_name', help='The name of the share.')
+
+    with self.argument_context('datashare share synchronization list-detail') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('account_name', help='The name of the share account.')
+        c.argument('share_name', help='The name of the share.')
+        c.argument('skip_token', help='Continuation token')
+        c.argument('consumer_email', help='Email of the user who created the synchronization')
+        c.argument('consumer_name', help='Name of the user who created the synchronization')
+        c.argument('consumer_tenant_name', help='Tenant name of the consumer who created the synchronization')
+        c.argument('duration_ms', help='synchronization duration')

```

Comment:
i think CLI guide is not put unit in argument name but in help message

Reply:
property removed in request.